TemplateProject/ios/TemplateProject.xcodeproj/project.pbxproj

```diff
@@ -504,7 +504,7 @@
 			isa = PBXProject;
 			attributes = {
 				LastUpgradeCheck = 610;
-				ORGANIZATIONNAME = Facebook;
+				ORGANIZATIONNAME = CompanyName;
```

Comment:
I think this will break if the company name the user enters contains a space. Maybe wrap this in quotes?